src/PowerShell/scripts/Execute-WinGetTests.ps1

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+[CmdletBinding()]
+param(
+    # The version of the client PS module to use.
+    [string]$ClientModuleVersion,
+
+    # The version of the configuration PS module to use.
+    [string]$ConfigurationModuleVersion,
+    
+    # The version of the client to use. Use 'existing' to skip updating.
+    [string]$ClientVersion,
+    
+    # The version of Powershell to use. Use 'existing' to skip updating.
+    [string]$PwshVersion,
+    
+    # The path to the binaries to run the web server.
+    [string]$LocalhostWebServerPath = (Join-Path $PSScriptRoot "..\LocalhostWebServer"),

```

Comment:
For any parameters which are paths, it may be a good idea to validate the path exists as part of the parameter validation
```powershell
[ValidateScript({if ($_){  Test-Path $_}})]
```